cypress/e2e/ContactUs.cy.js

```diff
@@ -31,4 +32,9 @@ describe('Validate Contact Us Landing page', () => {
       .should('be.visible')
       .and('have.text', 'Mon tableau de bord')
   })
+
+  it.skip('Validate the EI link navigates to EI Contact us page', () => {
```

Comment:
Just a question, why do we skip tests sometimes?

Reply:
> Just a question, why do we skip tests sometimes?

some times I just create templates or create tests based on the wireframes but code is still in development. so I skip those test cases